config/sync/node.type.frontpage.yml

```diff
@@ -0,0 +1,18 @@
+uuid: 01e7fae7-ccaf-4161-8e52-9fc9a78d12b5
```

Comment:
Although the settings are mostly correct, I'd rather recreate this Content Type with a different name. There's a saying that's been around for a while:

> There are only two hard things in Computer Science: cache invalidation and naming things.
>
> -- Phil Karlton

Think about Content Types as templates for storing data. Templates are meant to be reused, sites usually have only one front page so this is a rather unfortunate naming. I'd suggest using `Landing page` as a content type because that can be reused for multiple purposes.